manifests/charts/gateway/templates/service.yaml

```diff
@@ -15,12 +15,14 @@ spec:
 {{- with .Values.service.loadBalancerIP }}
   loadBalancerIP: "{{ . }}"
 {{- end }}
-{{- with .Values.service.ipFamilyPolicy }}
-  ipFamilyPolicy: "{{ . }}"
+{{- if .Values.service.ipFamilyPolicy }}
+  ipFamilyPolicy: {{ .Values.pilot.ipFamilyPolicy }}
```

Comment:
does this mean that we copy the value from the pilot config for gateway config, if not explicitly specified?

Reply:
No, this is a bug. I'll fix it.